### DISCUSSION TITLE
Absent logic on solving Public Goods Problem [Web3 / Blockchain] in "Transcending the Individualist/Collectivist Divide"

### DISCUSSION THREAD

**Discussion by rufuspollock** [rufuspollock/awesome-crypto-critique#72] [General]
I was pointed at this thread / deck:  https://twitter.com/owocki/status/1458541358147256322 + https://docs.google.com/presentation/d/1Q4X8Z9CTptbVwbw2SRHKHFHUBE1NV40BKx1_MvTUIcs/edit#slide=id.g1012330f0de_2_38

The intro states:

> I wrote a talk about how we can fund public goods by transcending the differences between Individualism  & Collectivism.

<img src="https://user-images.githubusercontent.com/180658/145490564-944cc661-e228-4bf5-bee9-7f661aab6281.png" height="250" />

## Analysis

After some fairly high-level analysis about individualism and collectivism the key claims on the actual funding for public goods (the "hard problem") are slide 12 and 17. After several read throughs I still couldn't identify any justification for the major claim that web3 addresses/solves the funding problem. Just a statement that one can "stack DAOs". **In particular, there's no discussion afaict of how the free rider problem is addressed which is *the* key issue here**. What happens if people who use the public good don't contribute? And if nothing happens, how do you avoid free-riding leading to the under-funding/under-contribution we so obviously see today? And if something does happen doesn't that involve some kind of exclusion (e.g. reversion to club good) or coercion?

Slide 12

<img src="https://user-images.githubusercontent.com/180658/145487315-fa520a7c-840a-4d92-900d-85c1d20fbd4e.png" height="650" />

**Comment**: hmmm, where's the justification in this deck that the classic "public goods" quadrant are web3 fundable?

5 slides on (see below) one simply has the statement "We can fund public goods at scale by stacking Impact DAOS."

How?? The key problem in funding public goods that is behind states for the last few thousand years is "show me the money"! How do you get people to contribute money *voluntarily* at *scale* given the *free-rider problem*.

Just consider the basics of living in a shared house and doing the dishes or managing the fridge? The free-rider problem is a serious issue at some point. And how does it get solved (usually): by defined and *enforced* contribution (or departure).

More broadly for society, and put crudely, philanthropy (optional contribution by the rich) doesn't cut it. Do the basic math: most modern states use ~40% of GDP. That's a lot of money - are people going to voluntarily contribute that?? (Sure, not all of gov spending is public goods - a bunch is service provision but a good portion is public goods).

And for governance why is a DAO then any more attractive than a democracy with the rule of law?

Slide 17/18:

![image](https://user-images.githubusercontent.com/180658/145487633-4e8bc2de-be84-4928-a03b-45ae8620e813.png)

![image](https://user-images.githubusercontent.com/180658/145487429-b1c99f55-c851-4190-8ccb-1fa7e316e0b7.png)

**Comment** How? Where does the money come from?

If membership of the DAO is a boundary how is this any different from "club goods" (aka making public goods partially excludable e.g. the membership based swimming pool / golf course etc)?

## Backups

![image](https://user-images.githubusercontent.com/180658/145487290-25adf0a6-3941-4054-8c91-512ab2943dc7.png)

![image](https://user-images.githubusercontent.com/180658/145487122-46634b0a-ff7a-4e65-8a12-ad0638e729f6.png)

**Comment by rufuspollock** [comment]
Thread with the author from the google presentation:

### Rufus

How? Where does the money come from? How is this any different from "club goods" (aka making public goods partially excludable e.g. the membership based swimming pool / golf course etc)?

### Kevin Owocki

https://gov.gitcoin.co/t/where-does-the-gitcoin-grants-matching-pool-money-come-from/9036

### Rufus Pollock

Right, so people "donate", mainly from the giant bubble that is crypto atm. How does a (relatively) small amount of funding from a tech bubble get us close to implying that one can fund (most) public goods via web3? (Let's just take global gov funded R&D which is billions/trillions a year ...)

### Kevin Owocki

there were plenty of donations worth millions of $$$ before the recent run up in crypto prices.

quadratic funding is more a efficient & effective capital allocation mechanism than what is used in most nation-state governments to support local public goods (large bureacracies), so its not hard to imagine it being deployed in a more scaled environment. this article in the economist today kind of covers that. https://twitter.com/glenweyl/status/1468993656963043332?s=21

to be clear, i dont think this is a guarantee and if you've already made up your mind i don't feel a strong urge to try and convince you otherwise - i just think it's one way things could unfold.

### Rufus Pollock

The key question is how do you deal with the free rider problem at scale. What you have confirmed is that you are essentially running on donations. Whilst the level of those donations may be somewhat extraordinary (and surely largely based on the crypto bubble of the last 5y) it is still fractions of a percent of the scale we'd want. It was like when people use to argue the kickstarter or similar would solve this problem.

The point about quadratic funding is a side-point surely - that's about governance for distribution (as you acknowledge). Even if one bought the efficiency of that (which I don't that much esp at scale and complexity) I'm doubtful that is real obstacle to people contributing - it's about the free-rider problem etc.

BTW this isn't me criticising your effort - or other less crypto-oriented ones such as opencollective. I think it is lovely you are doing this. What I'm taking issue with is the wild claims that we've somehow solved the public goods problem.

NB: that's not an article in the economist but a letter to the editor from the person who proposed quadratic funding.

### Kevin Owocki

> What I'm taking issue with is the wild claims that we've somehow solved the public goods problem.

who is making those claims? i certainly disagree with whomever has said that.

### Kevin Owocki

> What you have confirmed is that you are essentially running on donations.

in a world where there are DEFI protocols that are a base substrate of the open financial system have pledged an ongoing percentage of their protocol value to public goods, i disagree that is "running on donations". this is already something that yearn + a few other DEFI protocols have done.

i feel like youre making presumptions and putting words in my mouth, and that makes me consider that maybe this discussion is not worth my time. i dont owe you anything, and i've got nothing to prove to you -- especially if you are going to continue hearing what you want to hear. if you want to continue the discussion you might consider being a little bit more open minded or approaching it with a beginners mind.

### Rufus Pollock

> who is making those claims? i certainly disagree with whomever has said that.

"We can fund public goods at scale by stacking Impact DAOS." (in this slide)

"BUT Public goods - Goods that are non-excludable & non-rivalrous - I believe they can be funded by web3." (in twitter thread).

### Rufus Pollock

> i feel like youre making presumptions and putting words in my mouth, and that makes me consider that maybe this discussion is not worth my time. i dont owe you anything, and i've got nothing to prove to you -- especially if you are going to continue hearing what you want to hear. if you want to continue the discussion you might consider being a little bit more open minded or approaching it with a beginners mind.

I'm sorry if you feel got at - i reiterate my last comment of appreciate the overall effort by you and others to fund more public goods. Plus that you are doing something and are "on the court" (vs the stands) is great 👏 and i'm sure you take your fair share of critique. I can also take a deep breath 😉

Coming from the heart, why I get passionate (aka re-active) here is that:

a) I personally really care about this problem: i.e. how we fund public goods especially informational ones. I've spent years looking at it and even written book about how we could solve it https://openrevolution.net with a focus on Remuneration Rights (https://openrevolution.net/remuneration-rights).

b) (IMO) many of the best minds and lots of resources are diverted into fancy tech/governance hacks (that won't work) rather than the hard, long-term effort to achieve political and (old-school) institutional change. The result is we don't get the change we want / need.

But I can get less reactive / definitive about that 😄

> in a world where there are DEFI protocols that are a base substrate of the open financial system have pledged an ongoing percentage of their protocol value to public goods, i disagree that is "running on donations". this is already something that yearn + a few other DEFI protocols have done.

OK. It would help to actually run those numbers & put them in the deck e.g. X% with Y protocol value => Z funding. And compare that to current funding by great cooperatives known as democratic nation states.

**Reply by AndreasS2501** [reply]
Just so you know, these public thought dumps are actually useful to other people. I'm maintaining myself a zettelkasten. Generally I shy away from sharing too much, because I fear the problem of context or misinterpretation. I found the life itself podcast through Joe Lightfoots - LiminalWeb blog post. I will tune in for a while and see where there might be interesting spaces for discussions.

**Reply by rufuspollock** [reply]
@AndreasS2501 that's really good to hear and thanks for taking the time to share that with us - it makes a difference and encourages us to do more. If you are interested there is plan to turn this repo into a full-fledged zettelkasten / digital garden https://github.com/life-itself/community/issues/89

**Reply by AndreasS2501** [reply]
I just saw that you did a session on the stoa, I plan to watch that this month. Eventually I hope to reach out to you and we could have a conversation about further possible collaboration. I'm really looking forward to that.

**Reply by rufuspollock** [reply]
@AndreasS2501 sounds great - if you want to contribute to web3 stuff check out our new project https://github.com/life-itself/web3 - https://web3.lifeitself.us/

**Reply by AndreasS2501** [reply]
Hi @rufuspollock , I took a look at your web3 notes, I also watched the stoa session. I did see that you made 2 videos with Stephen Diehl.
I think I would like to talk with you on the web3 topic.  

Where could I schedule some time ( 1h?) with you in that regard? I don't want to put in too much context here, but for example I am part of the awakening from the meaning crisis (by john verveake) discord. But I also know the web3 space through things like kernel0x. So I'm really looking forward to discuss things with you.

**Reply by rufuspollock** [reply]
Sure. Drop a line via https://rufuspollock.com/contact/

**Reply by AndreasS2501** [reply]
Hi @rufuspollock I sent you a mail. I hope we can find some time together.
